conf/defaults.ini

```diff
@@ -483,10 +483,6 @@ verify_email_enabled = false
 # Redirect to default OrgId after login
 login_default_org_id =
 
-# Background text for the user field on the login page
-login_hint = email or username
-password_hint = password
-
```

Comment:
We should leave these in here, to indicate that they _can_ be set by the user, but by default they're empty. We want to preserve the same functionality.